### PR TITLE
Update user guide

### LINKED ISSUE #127: New parsing structure renders command sections of User Guide obsolete
User Guide should be updated to match the new input structure. Severity is high as new users would not be able to use the app.

### PR DESCRIPTION
**Pull request by fyshhh**
restructured user guide and updated pictures. closes #127

### CODE REVIEW

**Review by solkim-83 — APPROVED** [review]
looks much more organised. thanks for doing it up.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,34 +3,74 @@ layout: page
...
+1. Ensure you have Java `11` or above installed in your computer.
 
-1. Download the latest `Athena.jar` (no releases yet).
+1. Download the latest `Athena.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-4/tp/releases).
 
 1. Copy the file to the folder you want to use as the _home folder_ for Athena.
 
-1. Double-click the file or run it with `java -jar Athena.jar` to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file or run it with `java -jar Athena.jar` to start the app. The GUI similar to the below should
+   appear in a few seconds. Note how the app contains some sample data.<br>
```
Comment (chan-j-d) [inline docs/UserGuide.md]: "A window similar to the one below should..." instead of "The GUI similar to the below should..."

**Review by chan-j-d — CHANGES_REQUESTED** [review]
Just one minor issue spotted

**Review by chan-j-d — COMMENTED** [review]
LGTM!

**Review by chan-j-d — APPROVED** [review]
LGTM!

**Pull request merged**